README.md

```diff
@@ -139,17 +139,27 @@ This will create a file in the root directory, e.g ```adp_gamma_dump.psql.gz```
 
 To restore this file to one of the other environments, type:
 
-```
-./script/db_upload.rb <ssh-name> <environment> [<ip_address>] filename
+```bash
+$ ./script/db_upload.rb <ssh-name> <environment> [<ip_address>] filename
 ```
 
 In this case, ```environment``` CANNOT be gamma.
 
 
 To load the database dump on to your local database, use:
 
+```bash
+$ rake db:restore[dump-file]
 ```
-rake db:restore[dump-file]
+
+Snippet for local dump and restore:
+
+```bash
+$ cd <cccd_root>
+$ ./script/db_dump.rb <sshusername> gamma <IP|knownhost>
+$ gunzip adp_gamma_dump.psql.gz
```

Comment:
ah, thought i tried it but i think i had given it the wrong file name on that attempt